wp-content/themes/twentytwenty-child/js/scripts.js

```diff
@@ -18,23 +18,125 @@ const bc_select_tab = ($, page_num) => {
  * creates event listeners for each tab click
  * @param $ - jQuery Selector
  */
-const bc_tabs = ($) => {
+const bind_tabs_event = ($, form_id) => {
 
-  // BC Tab 1: Your Information
-  $('#gf_step_4_1').live('click', () => bc_select_tab($, '1'));
+  // Loop through the 3 tabs and set up event listeners
+  for (let page_num = 1; page_num <= 3; page_num++) {
+    $(`#gf_step_${form_id}_${page_num}`)
+      .attr('role', 'button')
+      .attr('aria-pressed', 'false')
+      .attr('tabindex', page_num)
+      .live('click', () => add_tab_select($, form_id, page_num));
+  }
 
-  // BC Tab 2: Contact
-  $('#gf_step_4_2').live('click', () => bc_select_tab($, '2'));
+}
+
+// STANDARD-SIZE PRINTING FORM
+
+/**
+ * Business Card Form Setup
+ *
+ * runs functions to set up business card form
+ * @param $
+ */
+const bc_form_setup = ($, form_id) => {
+
+  // add tab functionality to business cards
+  bind_tabs_event($, form_id);
+
+}
+
+/**
+ * Standard-Size Printing Form Adding Classes
+ *
+ * current workaround to add classes to WooCommerce product forms
+ * (gf classes are removed when forms are integrated with WC)
+ * @param $ - jQuery selector
+ * @param form_id
+ */
+const ssp_form_class = ($, form_id) => {
+
+  //@TODO - id selector is not working, will need to resolve later on
+
+  // Add "standard-size-form" & "cc-form" class
+  $(`form.cart`)
+    .addClass('standard-size-form')
+    .addClass('cc-form');
+
+}
+
+/**
+ * SSP Conditional Checkbox Choices
+ *
+ * only allow for one type of staple to be chosen for stapling
+ * @param $
+ * @param form_id
+ */
+const ssp_staple_conditional_checkbox = ($, form_id) => {
+
+  if (form_id === 1) {
+
+    // array of child inputs
+    // const checkboxInputs = $(`li.gfield.staple.hole-punch div.ginput_container li input`);
+
+    const one_staple = $('input#choice_1_66_1');
+    const two_staple = $('input#choice_1_66_2');
+    const four_staple = $('input#choice_1_66_3');
+
+    // if 1-staple is chosen, remove options 2- & 4-
+    one_staple.change(() => {
```

Comment:
Maybe we can use the same type of conditionals for that bug we're having when selecting on SSP in relation to the different level options.